src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+@use "../utils/_variables" as *;
+
+.main {
+  display: flex;
+  max-width: 944px;
```

Comment:
the checklist, p.13

Reply:
Paul said that p13 does not neccessary now